### PR TITLE
feat(samples): add create-featurestore sample

### PR DESCRIPTION
**Pull request by sai-chaithu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

#945

### CODE REVIEW

File: samples/snippets/src/main/java/aiplatform/CreateFeaturestoreSample.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    int minNodeCount = 1;
+    int maxNodeCount = 5;
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 900;
```
Comment (diemtvu) [inline samples/snippets/src/main/java/aiplatform/CreateFeaturestoreSample.java:51]: Pls change this 15 minutes

Comment (sai-chaithu) [reply samples/snippets/src/main/java/aiplatform/CreateFeaturestoreSample.java:51]: The timeout unit is kept in seconds so I have given 900 which is equal to 15 minutes,  Should I change the timeout unit to minutes??

Comment (diemtvu) [reply samples/snippets/src/main/java/aiplatform/CreateFeaturestoreSample.java:51]: Ah, sorry my math. lgtm

**Comment by diemtvu** [comment]
Please move new changes to a new PR. Let's keep this PR small so to make sure all reviews are done properly. Thanks

**Review by diemtvu — DISMISSED** [review]

**Review by diemtvu — APPROVED** [review]

**Review by dizcology — APPROVED** [review]

**Pull request merged**